src/theory/datatypes/datatypes_sygus.h

```diff
@@ -117,29 +124,140 @@ class SygusSymBreakNew
     /** For each term, whether this cache has processed that term */
     std::map< Node, bool > d_search_val_proc;
   };
-  // anchor -> cache
+  /** An instance of the above cache, for each anchor */
   std::map< Node, SearchCache > d_cache;
   /** a sygus sampler object for each (anchor, sygus type) pair
    *
    * This is used for the sygusRewVerify() option to verify the correctness of
    * the rewriter.
    */
   std::map<Node, std::map<TypeNode, quantifiers::SygusSampler>> d_sampler;
-  Node d_null;
+  /** Assert tester internal
+   *
+   * This function is called when the tester with index tindex is asserted for
+   * n, exp is the tester predicate. For example, for grammar:
+   *   A -> A+A | x | 1 | 0
+   * when is_+( d ) is asserted, assertTesterInternal(0, d, is_+( d ),...) is
+   * called. This function may add lemmas to lemmas, which are sent out on the
+   * output channel of datatypes by the caller.
+   */
   void assertTesterInternal( int tindex, TNode n, Node exp, std::vector< Node >& lemmas );
```

Comment:
Clarify what this function actually does, what are the lemmas it may add to the output channel.